modules/ROOT/pages/service-mesh-overview-and-landing-page.adoc

```diff
@@ -15,30 +15,33 @@ To alleviate issues arising from differences in these microservices, Anypoint Se
+The mixer is a highly modular and extensible component, whose key function is to provide a layer of abstraction over the different policy and telemetry backend systems. This enables applications, such as Anypoint Platform to allow the rest of Istio to be agnostic of the backend processing.
+
+The mixer is architected on a general-purpose plug-in model, which provides flexibility in dealing with different infrastructure backend applications. These plug-ins applications, known as adapters, enable the mixer to interface with different infrastructure backend applications that deliver core functionality, such as logging, monitoring, enforcing quotas, performing ACL checks, and so on.
 
 image::anypoint-service-mesh-architecture.png[Anypoint Service Mesh Architecture]
 
-As illustrated in the diagram, at the top of the architecture is Anypoint Platform, which brings complete lifecycle API management capabilities to the microservices built with Mule Runtime engine. These Mule Runtimes might be deployed within CloudHub, or a Runtime Fabric at your location. 
+As illustrated in the diagram, at the top of the architecture is Anypoint Platform, which brings complete lifecycle API management capabilities to the microservices built with Mule runtime engine (Mule). These Mule instances might be deployed within CloudHub, or Runtime Fabric at your location. 
 
 Istio, which is installed on a Kubernentes cluster, uses Envoy to manage the services using a sidecar proxy. The sidecar is responsible for any communication with the service, and resides within the same container as your service. With the Anypoint Service Mesh enabled, you can easily apply any of Istio’s native policies for traffic control and security. These policies help ensure reliable and secure service-to-service communication.
 
 The MuleSoft adapter and broker, which reside within the Istio mixer, connect components to Anypoint Platform. The adapter and broker enable sharing of metadata about all the services that are managed by the mesh. This allows for discovery of your existing microservices from the Kubernetes cluster as APIs into Anypoint Platform. 
 
-Anypoint Exchange then uses the metadata to automatically discover and create HTTP APIs for each service. These APIs can then be managed within API Manager. Similarly, Anypoint Monitoring uses the metadata to get information from the sidecar proxy and makes it available for API Analytics. 
+After the services are discovered, Anypoint Exchange then uses the metadata to automatically discover and create HTTP APIs for each service. These APIs can then be managed within API Manager. Similarly, Anypoint Monitoring uses the metadata to get information from the sidecar proxy and makes it available for API Analytics.
```

Comment:
Anypoint Exchange _then_ uses > Anypoint Exchange uses

Reply:
not auxiliary. Users understand sidecar as seen in Beta.